packages/site/src/routes/[dictionaryId]/contributors/Citation.svelte

```diff
@@ -53,6 +54,6 @@
     {citation?.citation ? citation.citation + ' ' : ''}
     {new Date().getFullYear()}.
     <span>{$page.data.t('dictionary.full_title', { values: { dictionary_name: dictionary.name }})}.</span>
-    Living Tongues Institute for Endangered Languages. https://livingdictionaries.app/{dictionary.id}
+    Living Tongues Institute for Endangered Languages{partners?.length ? ', ' + partners.map(partner => partner.name).join(', ') : ''}. https://livingdictionaries.app/{dictionary.id}
```

Comment:
@Danble This sort of situation which is just text only should not be tested via a variant, but rather via a simple set of unit tests. Much lighter and faster to run and easier to quickly mock different situations. It is fine for now but if you make any further changes to the citation in a future PR let's create a function that receives all the possible inputs and outputs the citation. That will be easy to test.